### PR TITLE
Standardise GA events

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This makes our GA events labels uniformly named across the whole codebase, events discrepancies [has been address in a separate PR](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1342), except one `question-error` event that is replaced with `error` event in this PR. "Open Panel" events dynamic nature has been removed and replaced with a hardcoded version as it wasn't very consistent and informative before. For the question and answer error events we need to come up with a  different structure and it is a potential bigger future change so I left them untouched.

### How to review
Compare with our documentation, check if the new wording follows the pattern. I held off with updating the events table in our doc as it's being used by other teams and the events labels text is not approved in this PR yet.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by rmccar — APPROVED** [review]

File: templates/layouts/_base.html [hunk 1]
```diff
@@ -14,7 +14,8 @@
         "attributes": {
           "data-ga": 'click',
           "data-ga-category": 'Navigation',
-          "data-ga-action": 'Previous link click'
+          "data-ga-action": 'Previous',
+          "data-ga-label": "Previous link click"
```
[the remark this answers is not in the document]
Comment (petechd) [reply templates/layouts/_base.html]: I've added them everywhere now, I can't see djlint changing them in the other PR.

[the remark this answers is not in the document]
Comment (rmccar) [reply templates/layouts/_base.html]: Ok cool, I cant see that either. Not having them on the last one is a nunjucks thing but I don't think it matters in jinja, I was just thinking we should be consistent either way

[the remark this answers is not in the document]
Comment (rmccar) [reply templates/layouts/_base.html]: Seems like there are still some missing

[the remark this answers is not in the document]
Comment (petechd) [reply templates/layouts/_base.html]: Ok, I think I've added them all now.

**Comment by ons-eq-team** [comment]
**Benchmark Results**<br />Percentile Averages:<br />50th: 60ms<br />90th: 103ms<br />95th: 180ms<br />99th: 320ms<br />99.9th: 511ms<br />GETs (99th): 390ms<br />POSTs (99th): 239ms<br />Total Requests: 70,576<br />Total Failures: 0<br />Error Percentage: 0.0%

**Comment by ons-eq-team** [comment]
---Percentile Averages:50th: 120ms90th: 277ms95th: 318ms99th: 451ms99.9th: 459ms---GETs (99th): 613msPOSTs (99th): 263ms---Total Requests: 6,556Total Failures: 0Error Percentage: 0.0%

**Comment by ons-eq-team** [comment]
**Benchmark Results**<br />---</ br>Percentile Averages:</ br>50th: 120ms</ br>90th: 277ms</ br>95th: 318ms</ br>99th: 451ms</ br>99.9th: 459ms</ br>---</ br>GETs (99th): 613ms</ br>POSTs (99th): 263ms</ br>---</ br>Total Requests: 6,556</ br>Total Failures: 0</ br>Error Percentage: 0.0%</ br>

**Comment by ons-eq-team** [comment]
**Benchmark Results**<br /><br />Percentile Averages:<br />50th: 120ms<br />90th: 277ms<br />95th: 318ms<br />99th: 451ms<br />99.9th: 459ms<br /><br />GETs (99th): 613ms<br />POSTs (99th): 263ms<br /><br />Total Requests: 6,556<br />Total Failures: 0<br />Error Percentage: 0.0%<br />

**Comment by ons-eq-team** [comment]
**Benchmark Results**<br />Percentile Averages:<br />50th: 120ms< br/>90th: 277ms< br/>95th: 318ms< br/>99th: 451ms< br/>99.9th: 459ms<br />GETs (99th): 613ms<br />POSTs (99th): 263ms<br />Total Requests: 6,556<br />Total Failures: 0<br />Error Percentage: 0.0%<br />

**Comment by ons-eq-team** [comment]
**Benchmark Results**<br />Percentile Averages:<br />50th: 120ms< br />90th: 277ms< br />95th: 318ms< br />99th: 451ms< br />99.9th: 459ms<br />GETs (99th): 613ms<br />POSTs (99th): 263ms<br />Total Requests: 6,556<br />Total Failures: 0<br />Error Percentage: 0.0%<br />

**Review by berroar — APPROVED** [review]
Seems to match the agreed changes 👍

**Pull request merged**